routers/web/web.go

```diff
@@ -575,9 +575,9 @@ func registerRoutes(m *web.Route) {
 			m.Combo("/new").Get(admin.NewUser).Post(web.Bind(forms.AdminCreateUserForm{}), admin.NewUserPost)
 			m.Get("/{userid}", admin.ViewUser)
 			m.Combo("/{userid}/edit").Get(admin.EditUser).Post(web.Bind(forms.AdminEditUserForm{}), admin.EditUserPost)
-			m.Post("/{userid}/delete", admin.DeleteUser)
-			m.Post("/{userid}/avatar", web.Bind(forms.AvatarForm{}), admin.AvatarPost)
-			m.Post("/{userid}/avatar/delete", admin.DeleteAvatar)
+			m.Post("/{userid}/edit/delete", admin.DeleteUser)
```

Comment:
You should change template link but not the routers I think.

Reply:
Done [`3beab41` (#27007)](https://github.com/go-gitea/gitea/pull/27007/commits/3beab41b8f5c064babaeb004e18e02d4792cce33)

Reply:
No, do not do this `ctx.Data["Link"] = "/admin/users/" + ctx.Params(":userid")` , it is wrong.